LanguageFeatures/Parts-with-imports/export_A01_t01.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Any Dart file can contain an `export` directive. It makes no
+/// difference which file an `export` is in, its declarations (filtered by any
+/// `hide` or `show` modifiers) are added to the library’s single export scope,
+/// along with those of any other `exports` in the library and the non-private
+/// declarations of the library itself.
+///
+/// @description Check that export directive contained in a part files of a
+/// library is added to the library's export scope.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=enhanced-parts
+
+import 'export_A01_t01_lib.dart';
+import '../../Utils/expect.dart';
+
+main() {
+  // From parts_lib.dart
```

Comment:
It's great to have a comment like this, thanks! In general, I think it's somewhat hard to read code where the import/export graph plays an important role, because readers have no easy way to get an overview of all the files and their connections.